README.md

```diff
@@ -279,7 +279,7 @@ python setup.py develop
 **On macOS**
 
 ```bash
-python3 setup.py develop
+USE_CUDA=0 USE_MKLDNN=0 USE_DISTRIBUTED=1 python3 setup.py develop
```

Comment:
These should already be the default no?

Reply:
Adding the following to CMakeLists.txt in the repository root:

```
if(NOT ${USE_CUDA} EQUAL "0")
  message(FATAL_ERROR "USE_CUDA should be 0")
endif()

if(NOT ${USE_MKLDNN} EQUAL 0)
  message(FATAL_ERROR "USE_MKLDNN should be 0")
endif()

if(NOT ${USE_DISTRIBUTED} EQUAL 1)
  message(FATAL_ERROR "USE_DISTRIBUTED should be 1")
endif()
```

confirms that this is already the default. I'll close this PR and dig deeper to see what lead to my unusable build.